bin/export-from-har.js

```diff
@@ -0,0 +1,111 @@
+const noop = function () {};
+const { JSDOM, ResourceLoader } = require( 'jsdom' );
+class Window {
+	constructor( jsdomConfig = {} ) {
+		const { proxy, strictSSL, userAgent } = jsdomConfig;
+		const resources = new ResourceLoader( {
+			proxy,
+			strictSSL,
+			userAgent,
+		} );
+		return new JSDOM(
+			'',
+			Object.assign( jsdomConfig, {
+				resources,
+			} )
+		).window;
+	}
+}
+
+global.window = new Window( { url: 'http://localhost' } );
+global.document = window.document;
+global.requestAnimationFrame = global.cancelAnimationFrame = noop;
+global.navigator = window.navigator;
+global.Mousetrap = {
+	init: noop,
+	prototype: { stopCallback: noop },
+};
+window.matchMedia = global.matchMedia = () => ( { addListener: noop } );
+global.Node = window.Node;
+
+const { registerCoreBlocks } = require( '@wordpress/block-library' );
+
+registerCoreBlocks();
+
+const fs = require( 'fs' );
+const fetchFromHAR = require( 'fetch-from-har' );
+
+const services = require( '../source/services' );
+const mockMapping = {
+	'manage.wix.com': {
+		'/_api/communities-blog-node-api/_api/posts': {
+			status: true,
+		},
+	},
+};
+
+const { Command } = require( 'commander' );
+const program = new Command();
+program.version( '1.0.0' );
+program
+	.command( 'wix' )
+	.option(
+		'-a, --appDefinitionId <appDefinitionId,...>',
+		'Which Wix module to extract',
+		[]
+	)
+	.option( '-m, --mediaToken <mediaToken>', 'Specify a media token', null )
```

Comment:
This option can be removed, since it doesn't need a real media token, just a value to insert into the config. (This can be confirmed by running it with `-m foo`, an invalid media token.)

Reply:
Removed